tests/cuda_test/test_kernel_semantic.py

```diff
@@ -203,7 +203,7 @@ def test_invalid_function_call(language):
     def invalid_function_call():
         def non_kernel_func():
             pass
-        non_kernel_func[1, 2]()
+        non_kernel_func[1, 2]() # pylint: disable=E1136
```

Comment:
Is the pylint command needed? The updated definition of the decorator should have fixed this

Reply:
This test checks the case where a non-kernel function is called with a defined launch configuration.Therefore, this command is necessary as the non-kernel function is not decorated with the kernel decorator.

Reply:
Ah of course sorry! :+1: